src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -479,11 +473,6 @@ namespace GraphQL.Builders
             where TEdgeType : GraphQL.Types.Relay.EdgeType<TNodeType>
             where TConnectionType : GraphQL.Types.Relay.ConnectionType<TNodeType, TEdgeType> { }
     }
-    public static class FieldBuilder
-    {
-        public static GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> Create<TSourceType, TReturnType>(GraphQL.Types.IGraphType type) { }
-        public static GraphQL.Builders.FieldBuilder<TSourceType, TReturnType> Create<TSourceType, TReturnType>(System.Type type = null) { }
-    }
```

Comment:
I understand that this "duplicates existing code", but I like `FieldBuilder.Create<>` better than `FieldBuilder<>.Create`.  Not a big deal though.

Reply:
One static method calls another one. It seems pointless to me.